packages/cache-ethereum-storage/src/cache-ethereum-storage.ts

```diff
@@ -0,0 +1,210 @@
+import { LogTypes, StorageTypes } from '@requestnetwork/types';
+import Utils from '@requestnetwork/utils';
+import * as Bluebird from 'bluebird';
+import { EventEmitter } from 'events';
+
+/**
+ * CacheEthereumStorage
+ * @notice Manages storage layer of the Request Network Protocol v2
+ */
+export default class CacheEthereumStorage implements StorageTypes.IStorage {
+  /**
+   * Logger instance
+   */
+  private logger: LogTypes.ILogger;
+
+  // TODO replace by database, files or something...
+  private data: { [id: string]: { content: string; timestamp: number } | null };
+
+  /**
+   * Url where can be reached the data hosted by this storage
+   */
+  private locationUrl: string;
```

Comment:
I don't understand this value, it is used only for meta ?
If revelant, should have a better description